lib/OpenQA/Schema/ResultSet/Assets.pm

```diff
@@ -21,16 +21,20 @@ sub status_cache_file {
 }
 
 # called when uploading an asset or finding one in scanning
+my $INSERT = 'INSERT INTO assets (type, name, t_created, t_updated) VALUES (?, ?, now(), now()) ON CONFLICT DO NOTHING';
```

Comment:
You don't really gain anything by declaring the query outside the function. Only makes it harder to follow the code imo.

Reply:
I gain that the line doesn't need to be wrapped but I suppose having it inline is generally better, indeed.